keywords/LiteServiOS.py

```diff
@@ -384,6 +392,27 @@ def stop(self):
             log_info("LiteServ is not running, so no need to stop")
         self._verify_not_running()
 
+    def _verify_running(self):
+        """
+        Return true if it is running or else false
+        Verifys that the endpoint does not return a 200 from a running service
```

Comment:
I don't we are verifying it, are we? Remove it if we are not

Reply:
I added logic to stop method , that is the reason I added